NGitLab/Models/JobCommon.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Runtime.Serialization;
+using System.Text.Json.Serialization;
+
+namespace NGitLab.Models
+{
+    [DataContract]
+    public class JobCommon
```

Comment:
Adding a base entity and moving fields and nested class is a binary breaking change. 

@jairbubbles Do you think this is a problem? If not, should we replace the fields by properties?

Reply:
I don't think it's too problematic, the jobs API is probably not used in many places. Yes, let's switch to properties in the meantime.